mqtt/mqtt-bridge/src/messages.rs

```diff
@@ -122,10 +128,20 @@ impl<S> StoreMqttEventHandler<S> {
     }
 }
 
+async fn push_publication<S>(
```

Comment:
Small helper fn I was working with while trying to make sure everything was at least Send. Can probably remove or extend this to be drop in below.